### DISCUSSION TITLE
Wiki Image Question

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2055] [Q&A] [answered]
@gwarser @uBlock-user @gorhill

I have a brief question in regards to this wiki image:

![46022246-3887c880-c0b0-11e8-9edc-ded22a011996](https://user-images.githubusercontent.com/95879668/158453805-5c17407f-cbb7-44a1-af04-19f4e7e18a8c.png)

What is the name of the special effect that is shown in this image? I mean the effect of the selected text being in focus / highlighted in lighter color as apposed to the background being darkened.

Can you also recommend any Linux screenshot tool or image editor that can perform this effect?

Thanks

**Comment by gorhill** [comment, marked as answer]
I use Gimp:

- Select the area to highlight
- Invert selection: Select => Invert
- Go to Colors => Brightness-Contrast... => Set brightness to -127 => Click OK

**Reply by ghost** [reply]
Hey @gorhill,

Thank you very much for the exact answer and how to do it! :smile: :+1:
